src/dotnet/Program.cs

```diff
@@ -114,6 +114,16 @@ internal static int ProcessArgs(string[] args, ITelemetry telemetryClient)
                 command = "help";
             }
 
+            var hashedargs = string.Join(",", args.Select(x => HashHelper.sha256_hash(x)));
```

Comment:
I think you want "appArgs" here, right?  Or is it ok to have the command inside the args again?